docs/guides/concurrent.md

```
@@ -138,9 +138,21 @@ async def notify_users():
     )
 ```
 
-### Executing with Results
+**Parameters:**
+- `source: AsyncIterable[Element] | Iterable[Element]` - Elements to process
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)